yt_dlp/extractor/radiofrance.py

```diff
@@ -56,8 +65,26 @@ def _real_extract(self, url):
         }
 
 
-class FranceCultureIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?radiofrance\.fr/(?:franceculture|fip|francemusique|mouv|franceinter)/podcasts/(?:[^?#]+/)?(?P<display_id>[^?#]+)-(?P<id>\d+)($|[?#])'
+class RadioFranceBase(InfoExtractor):
```

Comment:
Not a big deal, but
```suggestion
class RadioFranceBaseIE(InfoExtractor):
```
or
```suggestion
class RadioFranceBaseInfoExtractor(InfoExtractor):
```
is convention